src/ipcMain/index.ts

```diff
@@ -7,7 +7,11 @@ import IpcEvents from "../utils/IpcEvents";
 
 import "./updater";
 
-const DATA_DIR = join(app.getPath("userData"), "..", "Vencord");
+const DATA_DIR = process.env.VENCORD_USER_DATA_DIR ?? (
+    process.env.DISCORD_USER_DATA_DIR ?
+        join(process.env.DISCORD_USER_DATA_DIR, "..", "Vencord") :
```

Comment:
```suggestion
        join(process.env.DISCORD_USER_DATA_DIR, "..", "VencordData") :
```
I think this would be better cause otherwise it might conflict with Vencord source dir / be confusing